ui/apps/platform/src/Containers/DeferralConfiguration/VulnerabilitiesConfigurationContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+    Button,
+    Divider,
+    PageSection,
+    Split,
+    SplitItem,
+    Text,
+    Title,
+} from '@patternfly/react-core';
+
+function VulnerabilitiesConfigurationContainer() {
```

Comment:
nit: I think we could just say `VulnerabilitiesConfiguration` here. The `Container` part doesn't seem as beneficial